python/clx/tests/test_sequence_classifier.py

```diff
@@ -41,7 +41,7 @@ def test_train_model():
             y_train,
             learning_rate=3e-5,
             max_seq_len=128,
-            batch_size=32,
+            batch_size=6,
```

Comment:
For the future, let's try to avoid mixing tests for non-salient modules within the same PR.